docs/users-guide/Analyzing-and-Translating-Models.adoc

```diff
@@ -1452,6 +1452,23 @@ The JSON format is specified in the
 https://fprime.jpl.nasa.gov/devel/docs/reference/fpp-json-dict/[F Prime design
 documentation].
 
+When generating dictionaries with `fpp-to-dict`, the following types must be defined
+in FPP (either in the input model itself or in a dependency file):
```

Comment:
Let's say also that each of the defined types has to be an alias of an integer type.